pkgs/development/tools/build-managers/bmake/default.nix

```diff
@@ -89,15 +90,14 @@ stdenv.mkDerivation rec {
 
   setupHook = ./setup-hook.sh;
 
-  passthru.tests = {
-    bmakeMusl = pkgsMusl.bmake;
-  };
-
   meta = with lib; {
+    homepage = "http://www.crufty.net/help/sjg/bmake.html";
     description = "Portable version of NetBSD 'make'";
-    homepage    = "http://www.crufty.net/help/sjg/bmake.html";
-    license     = licenses.bsd3;
-    platforms   = platforms.unix;
-    maintainers = with maintainers; [ thoughtpolice ];
+    license = licenses.bsd3;
+    maintainers = with maintainers; [ thoughtpolice AndersonTorres ];
+    platforms = platforms.unix;
   };
+
+  passthru.tests.bmakeMusl = pkgsMusl.bmake;
 }
+# TODO: report the quirks and patches to bmake devteam (especially the Musl one)
```

Comment:
I think this sort of thing is better tracked in a GitHub issue — it's basically invisible here, especially at the bottom of the file.

Reply:
I would like to make this things a bit more, ehr, systematic. Many open source projects can benefit of those things found on Nixpkgs, especially those who seek portability like NetBSD ones.

Reply:
I agree for sure it would be better to be more systematic about patches and upstreams.  I think opening issues is a good start, because it's visible and can be commented on with updates in a single place.